components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/IdentityRecoveryConstants.java

```diff
@@ -215,6 +219,8 @@ private IdentityRecoveryConstants() {
         // UAV - User Account Verification.
         ERROR_CODE_UNSUPPORTED_VERIFICATION_CHANNEL("UAV-10001",
                 "Unsupported verification channel"),
+        ERROR_CODE_GETTING_VERIFICATION_PENDING_EMAIL("UAV-10002", "Error while getting verification " +
```

Comment:
This is thrown while getting the claim values from the userstore yeah? So shall we add that as well? WDYT?

Reply:
Improved the message